src/utils/command.rs

```diff
@@ -208,6 +209,9 @@ pub trait BotCommand: Sized {
     fn parse<N>(s: &str, bot_name: N) -> Result<Self, ParseError>
     where
         N: Into<String>;
+    fn ty() -> PhantomData<Self> {
```

Comment:
Why is this needed?

Reply:
CC @p0lunin

Reply:
Well, because we need to pass the type of command to `commands_repl`. If we used generics, it would be something like `commands_repl::<Cmd, _, _, _, _, _>`.